packages/example/src/App.tsx

```diff
@@ -10,6 +10,7 @@ import RawCardDataScreen from './screens/RawCardDataScreen';
 import RawPhoneNumberDataScreen from './screens/RawPhoneNumberScreen';
 import RawAdyenBancontactCardScreen from './screens/RawAdyenBancontactCardScreen';
 import RawRetailOutletScreen from './screens/RawRetailOutletScreen';
+import { HeadlessCheckoutVaultScreen } from './screens/HeadlessCheckoutVaultScreen';
```

Comment:
nit: This should be a default export / import to align with the other screens

Reply:
(Requires other suggestion below) 

```suggestion
import HeadlessCheckoutVaultScreen from './screens/HeadlessCheckoutVaultScreen';
```